types/selrond__knit/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
you have to update this file to support scoped package paths, as per readme:
https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages